javatests/arcs/sdk/wasm/WasmMultipleHandlesTest.kt

```diff
@@ -0,0 +1,177 @@
+package arcs.sdk.wasm
+
+class WasmMultipleHandlesTest : AbstractWasmMultipleHandlesTest() {
+    var singleHandleCount = 0
+    var tupleHandleCount = 0
+    var tripleHandleCount = 0
+    var quadHandleCount = 0
+    var quinHandleCount = 0
+    var sixHandleCount = 0
+    var septHandleCount = 0
+    var octHandleCount = 0
+    var novHandleCount = 0
+    var decHandleCount = 0
+
+    init {
+        handles.handle1.onUpdate{
+            singleHandleCount = singleHandleCount + 1
+        }
+
+        combine(
+                handles.handle1,
+                handles.handle2
+        ).onUpdate { _, _ ->
```

Comment:
I don't think you need to "name" the arguments at all. Can you just go `.onUpdate { tupleHandleCount++ }`?